integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -151,7 +151,10 @@ public Long load(String groupName) {
 
   private void setUserGroupIfNeeded(AlluxioURI uri) throws Exception {
     SetAttributePOptions.Builder attributeOptionsBuilder = SetAttributePOptions.newBuilder();
+    ByteBuffer buffer = this.getLibFuse().fuse_get_context();
     FuseContext fc = getContext();
+    // TODO(maobaolong): open it after PR12836 merged.
```

Comment:
@maobaolong  #12836 has been merged. you can update it now.

Reply:
OK